app/src/main/java/io/homeassistant/companion/android/sensors/SensorWorker.kt

```diff
@@ -27,11 +30,9 @@ class SensorWorker(private val appContext: Context, workerParams: WorkerParamete
             val sensorWorker =
                 PeriodicWorkRequestBuilder<SensorWorker>(15, TimeUnit.MINUTES)
                     .setConstraints(constraints)
-                    .addTag("sensors")
                     .build()
 
-            WorkManager.getInstance(context).cancelAllWorkByTag("sensors")
-            WorkManager.getInstance(context).enqueue(sensorWorker)
+            WorkManager.getInstance(context).enqueueUniquePeriodicWork(TAG, ExistingPeriodicWorkPolicy.REPLACE, sensorWorker)
```

Comment:
This change appeared to stop duplicate `SensorWorker`s from operating. I was noticing that when doing some debugging myself. The only odd thing here is that I used the `TAG` from the companion object. I could use "sensors" as was done previously if that is desired.